.github/workflows/cibuild.yml

```diff
@@ -152,7 +152,7 @@ jobs:
           # schema hasn't changed.
           # example commit: https://github.com/opentripplanner/docs/commit/45e6ddf8e4a4
           
-          SCHEMA_FILE_MODIFIED=`git log -n 1 --pretty=format:%ct src/ext/resources/gtfsgraphqlapi/schema.graphqls`
+          SCHEMA_FILE_MODIFIED=`git log -n 1 --pretty=format:%ct src/main/resources/gtfsgraphqlapi/schema.graphqls`
```

Comment:
Should the resource path be the same as the class package path? I'm not sure what is the convention.

Reply:
Personally I don't like to repeat the `org/opentripplanner/...` folder structure in resources but I know that there are other opinions. Lets discuss this.

Reply:
We will move it to `apis/gtfs/`.

Reply:
Since we decided that we want to move the resources into the `org/opentripplanner/...` hierarchy, it's now moved there.

Reply:
In Intellij's package view, this now looks like this:

![image](https://github.com/opentripplanner/OpenTripPlanner/assets/151346/144fb6ab-26e5-46ed-9505-5ffdba49eaad)

@t2gran